### PR TITLE
Add basic AVRO files (translated copies of the parquet testing files to avro)

### PR DESCRIPTION
**Pull request by Igosuki**
N.B. : I used spark for the translation so there is some additional metadata in the files, but they can be removed.

### CODE REVIEW

**Review by alamb — APPROVED** [review]

**Review by alamb — COMMENTED** [review]
@pitrou / @kiszk do you have any concerns or suggestions for adding several smaller AVRO files into the testing repository? They are used for https://github.com/apache/arrow-datafusion/pull/910 and we may consider adding avro support to the main apache-rs repo as well.

**Comment by pitrou** [comment]
This seems fine to me, but can you add a README explaining what these files are and how they were obtained?

**Review by houqp — APPROVED** [review]

**Comment by kiszk** [comment]
Looks good to me

**Comment by alamb** [comment]
@Igosuki  -- I added a basic README in https://github.com/apache/arrow-testing/pull/62/commits/8d306efa213b859645c68f60ee1f6db0b1997b43 -- can you provide the command you used to create these files from the original parquet?

Thanks!

**Comment by Igosuki** [comment]
@alamb https://gist.github.com/Igosuki/324b011f40185269d3fc552350d21744

**Comment by Igosuki** [comment]
It would be possible to use arrow-python and fastavro to achieve the same, I just have a lot of Spark experience and I prefer typed so I went that way.

**Comment by alamb** [comment]
Thanks @Igosuki ! I am sorry for the delayed response -- I am catching up from being on vacation and hope to help push your contributions over the line real soon now

File: data/avro/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+This directory contains AVRO files corresponding to the parquet testing files at https://github.com/apache/parquet-testing/blob/master/data/
+
+These files were created by using spark with the following commands:
+
+```
+TODO
```
Comment (alamb) [inline data/avro/README.md:6]:
```suggestion
These files were created by using spark using the commands from https://gist.github.com/Igosuki/324b011f40185269d3fc552350d21744

Roughly:
```scala
import com.github.mrpowers.spark.daria.sql.DariaWriters
import org.apache.hadoop.fs.FileSystem
import org.apache.hadoop.fs.Path
import org.apache.hadoop.conf.Configuration 
import org.apache.commons.io.FilenameUtils

val fileGlobs = sc.getConf.get("spark.driver.globs")
val dest = sc.getConf.get("spark.driver.out")

val fs = FileSystem.get(new Configuration(true));
val status = fs.globStatus(new Path(fileGlobs))
for (fileStatus <- status) {
    val path = fileStatus.getPath().toString()
    try {
        val dfin = spark.read.format("parquet").load(path)
        val fileName = fileStatus.getPath().getName();
        val fileNameWithOutExt = FilenameUtils.removeExtension(fileName);
        val destination = s"${dest}/${fileNameWithOutExt}.avro"
        println(s"Converting $path to avro at $destination")
        DariaWriters.writeSingleFile(
            df = dfin,
            format = "avro",
            sc = spark.sparkContext,
            tmpFolder = s"/tmp/dw/${fileName}",
            filename = destination
        )
    } catch {
        case e: Throwable => println(s"failed to convert $path : ${e.getMessage}")
    }
}
```

```

**Comment by Igosuki** [comment]
All good :+1:

**Pull request merged**